components/dashboard/src/teams/Members.tsx

```diff
@@ -295,6 +290,38 @@ export default function MembersPage() {
                     </ModalFooter>
                 </Modal>
             )}
+            {memberToRemove && (
+                // TODO: Use title and buttons props
+                <Modal visible={true} onClose={() => setMemberToRemove(undefined)}>
+                    <ModalHeader>Remove Members</ModalHeader>
+                    <ModalBody>
+                        You are about to remove <b>{memberToRemove.fullName}</b> from this organization.
+                        <br />
+                        <br />
+                        {memberToRemove.ownedByOrganization ? (
+                            <>This will delete the user account and all associated data.</>
+                        ) : null}
+                    </ModalBody>
+                    <ModalFooter>
+                        <Button variant="secondary" onClick={() => setMemberToRemove(undefined)}>
+                            Cancel
+                        </Button>
+                        <Button
+                            variant="default"
+                            onClick={async () => {
+                                await organizationClient.deleteOrganizationMember({
+                                    organizationId: org.data?.id,
+                                    userId: memberToRemove.userId,
+                                });
+                                invalidateMembers();
+                                setMemberToRemove(undefined);
```

Comment:
Everything happening till this point is 👍🏻 Can we add any next step here? Otherwise the state is in limbo. 
1. Select "Leave ..."
<img width="683" alt="Screenshot 2023-12-14 at 15 51 09" src="https://github.com/gitpod-io/gitpod/assets/914497/6754fdb5-d24d-465f-a9f2-9d4d8a6c4760">
2. Select "Remove"
<img width="881" alt="Screenshot 2023-12-14 at 15 51 15" src="https://github.com/gitpod-io/gitpod/assets/914497/7cf12664-ca43-42f0-9d95-ea9ef80da2b0">
3. **Now nothing happens.** 
4. If I retry 1) and 2) it throws some error, likely hidden as this is the dev mode, right?
<img width="1840" alt="Screenshot 2023-12-14 at 15 49 11" src="https://github.com/gitpod-io/gitpod/assets/914497/6ab7b637-4ae9-4bb8-ab5a-3c2ccc09c216">

Reply:
This is only affecting users leaving the Org. If the owner is removing members, the UX is fine.